app/vue/contexts/profile/ProfileDetailsPageContext.js

```diff
@@ -249,6 +271,56 @@ export default class ProfileDetailsContext extends BaseFuroContext {
     }
   }
 
+  /**
+   * Fetch profile orders.
+   *
+   * @param {{
+   *   address: string | null
+   * }} params - Parameters
+   * @returns {Promise<void>}
+   */
+  async fetchProfileOrders ({
+    address,
+  }) {
+    if (!address) {
+      return
+    }
+
+    try {
+      this.statusReactive.isLoadingProfileOrders = true
+
+      const headers = {
+        accept: 'application/json',
+      }
+      const searchParams = new URLSearchParams({
+        address,
+        parentSubaccountNumber: '0',
+        status: 'OPEN,UNTRIGGERED',
+      })
+      const resourceUrl = `${BASE_INDEXER_URL}/orders/parentSubaccountNumber?${searchParams.toString()}`
+
```

Comment:
@hecticme 

`try - catch` is not for blocking

It is good to include actual throwable code only

```suggestion
    this.statusReactive.isLoadingProfileOrders = true

    const headers = {
      accept: 'application/json',
    }
    const searchParams = new URLSearchParams({
      address,
      parentSubaccountNumber: '0',
      status: 'OPEN,UNTRIGGERED',
    })
    const resourceUrl = `${BASE_INDEXER_URL}/orders/parentSubaccountNumber?${searchParams.toString()}`

    try {
```